modules/serverless-logic-supporting-services-eventing-system-config.adoc

```diff
@@ -0,0 +1,252 @@
+
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-logic-supporting-services-eventing-system-config_{context}"]
+= Supporting services eventing system configurations
+
+This section describes the supporting services eventing system configurations.
+
+In {ServerlessLogicProductName} installation, the following types of events are generated:
+
+* Outgoing and incoming events related to workflow business logic.
+
+* Events sent from workflows to the Data Index and Job Service.
+
+* Events sent from the Job Service to the Data Index Service.
+
+The {ServerlessLogicOperatorName} leverages the Knative Eventing system to manage all event communication between these services, ensuring efficient and reliable event handling.
+
+[id="serverless-logic-supporting-services-platform-scoped-eventing-system-config_{context}"]
+== Platform-scoped eventing system configuration
+
+To configure a platform-scoped eventing system, you can use the `spec.eventing.broker.ref` field in the `SonataFlowPlatform` CR to reference a Knative Eventing Broker. This configuration instructs the {ServerlessLogicOperatorName} to automatically link the supporting services to produce and consume events using the specified Broker.
+
+Additionally, workflows deployed in the same namespace with the `preview` or `gitops` profile, and without a custom eventing system configuration, will automatically link to the specified Broker.
+
+[NOTE]
+====
+In production environments, use a production-ready broker, such as the Knative Kafka Broker, for enhanced scalability and reliability.
+====
+
+The following example displays how to configure the `SonataFlowPlatform` CR for a platform-scoped eventing system:
+
+.Example of platform-scoped eventing system configuration 
+[source,yaml]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform-example
+  namespace: example-namespace
+spec:
+  eventing:
+    broker:
+      ref:
+        name: example-broker <1>
+        namespace: example-broker-namespace <2>
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+----
+
+<1> Specifies the Knative Eventing Broker name.
+<2> Optional: Defines the namespace of the Knative Eventing Broker. If not specified, it defaults to the SonataFlowPlatform namespace. It is recommended to create the Broker in the same namespace as the SonataFlowPlatform.
+
+[id="serverless-logic-supporting-services-service-scoped-eventing-system-config_{context}"]
+== Service-scoped eventing system configuration
+
+A service-scoped eventing system configuration allows for fine-grained control over the eventing system, specifically for the Data Index or the Job Service.
+
+[NOTE]
+====
+In {ServerlessLogicProductName} installation, the recommended approach is to use a platform-scoped eventing system configuration. The service-scoped configuration is intended for advanced use cases only.
+====
+
+[id="serverless-logic-data-index-eventing-system-config_{context}"]
```

Comment:
Separate module?

Reply:
Kept it under the same module as the content was not that much and it is a part of the "eventing system".